apparelo/apparelo/doctype/grn/grn.py

```diff
@@ -36,54 +36,29 @@ def create_purchase_receipt(self):
 		pr.submit()
 		return pr
 	def get_po(self):
-		doc_=self.against_document
-		if doc_.startswith("D"):
-			all_po=frappe.db.get_all("Purchase Order")
-			for po in all_po:
-				po_=frappe.get_doc("Purchase Order",po.name)
-				if po_.dc==doc_:
-					doc_=po_.name
-		PO=frappe.get_doc("Purchase Order",doc_)
-		self.po=PO.name
-		return PO.name
+		dc_doc = frappe.get_doc("DC",self.against_document)
+		if dc_doc.doctype.startswith("D"):
```

Comment:
You `get_doc` a DC and check if it is a DC!

Reply:
> You `get_doc` a DC and check if it is a DC!

Minor mistake! will update soon.